lib/_http_server.js

```diff
@@ -726,7 +732,7 @@ function connectionListenerInternal(server, socket) {
   socket.setEncoding = socketSetEncoding;
 
   // We only consume the socket if it has never been consumed before.
-  if (socket._handle && socket._handle.isStreamBase &&
+  if (socket._handle.isStreamBase &&
```

Comment:
This is not the same. It will crash if `socket._handle` is `null` or `undefined.`